faq/how-do-i-edit-metadata.md

```diff
@@ -38,6 +38,7 @@ taxon:
   id: NCBITaxon:33208
   label: Metazoa
 domain: cells
+aberowl_id: ab
```

Comment:
I commented on the general approach of this PR in issue #1053, but my only specific problem with this PR is that I don't want `aberowl_id` in the basic example: I don't want people to use it unless they absolutely have to.

Reply:
agreed